### PR TITLE
remove glx mode since it no use anymore

### PR DESCRIPTION
**Pull request by xuguangxin**
glx mode original for chrome os, but it evolving from glx to ozone. So we'd better remove it.

### CODE REVIEW

**Comment by uartie** [comment]
`./autogen.sh --enable-v4l2 --enable-tests`
...
`../v4l2/.libs/libyami_v4l2.so: undefined reference to 'eglGetProcAddress'`

**Comment by uartie** [comment]
There's still a bunch of places in configure.ac that evaluate "$enable_v4l2_glx" that need removed... that might fix the compile error.

**Comment by xuguangxin** [comment]
done, it's so hard...

**Comment by uartie** [comment]
+1 lgtm

File: tests/v4l2decode.cpp [hunk 1]
```diff
@@ -207,10 +201,6 @@ static Display * x11Display = NULL;
 static Window x11Window = 0;
 #endif
 #ifdef ANDROID
```
Comment (chivakker) [inline tests/v4l2decode.cpp]: This is an empty #ifdef. consider removing it

**Comment by xuguangxin** [comment]
update according to Daniel's comments.

**Comment by halleyzhao** [comment]
+1
Thanks

**Comment by xuguangxin** [comment]
merged, thanks all.

**Pull request merged**